### PR TITLE
OKTA-537533 Added copyrights and trademarks page

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
- **What's changed?** Added copyrights and trademark acknowledgments page to the site footer 
- **Is this PR related to a Monolith release?** No

- Netlify link : https://6376476063ee3e1db1a92529--reverent-murdock-829d24.netlify.app/copyright/

### Resolves:

* [OKTA-537533](https://oktainc.atlassian.net/browse/OKTA-537533)

### CODE REVIEW

File: packages/@okta/vuepress-site/copyright/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+- Okta Headquarters North America<br>
+  100 First Street<br>
+  San Francisco<br>
+  CA 94105<br>
+  USA
+
+- Okta®, Auth0®, and the Okta® and Auth0® Logos are registered trademarks of Okta, Inc. in the U.S. and other countries.
+
+- Okta's Terms of Service for its Intellectual Property <a href="https://www.okta.com/terms-of-service/" target="_blank">can be found here</a>.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/copyright/index.md:25]: We avoid referencing links as "found here" or "here". Does this work instead?

```suggestion
- See <a href="https://www.okta.com/terms-of-service/" target="_blank">Okta's Terms of Service for its Intellectual Property</a>.
```

File: packages/@okta/vuepress-site/copyright/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- Google, Android™ platform, and Angular are trademarks or registered trademarks of Google LLC.
+
+- Amazon is a trademark or registered trademark of Amazon LLC.
+
+- Apple, the Apple logo, and iOS are trademarks of Apple Inc., registered in the United States and other countries.
+
+- Java is a registered trademark of Oracle.
+
+- Python is a registered trademark of the PSF. The Python logos (in several variants) are use trademarks of the PSF as well.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/copyright/index.md:39]:
```suggestion
- Python is a registered trademark of the PSF. The Python logos (in several variants) are trademarks of the PSF as well.
```

File: packages/@okta/vuepress-site/copyright/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+- Gitlab and the Gitlab logos (in several variants) are trademarks of Gitlab Inc.
+
+- LinkedIn, the LinkedIn logo, the IN logo, and InMail are registered trademarks or trademarks of LinkedIn Corporation and its affiliates in the United States and/or other countries.
+
+- Node.js and its wordmarks in plain text or logo form are registered trademarks or trademarks of the Node.js community.
+
+- OpenID® is a trademark (registered in numerous countries) of the OpenID Foundation.
+
+- Paypal, and the Paypal logo are registered trademarks or trademarks of Paypal, Inc.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/copyright/index.md:55]:
```suggestion
- Paypal and the Paypal logo are registered trademarks or trademarks of Paypal, Inc.
```

File: packages/@okta/vuepress-site/copyright/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+- The PHP logo is available under a Creative Commons Attribution-Share Alike 4.0 International license.
+
+- Salesforce, and the Salesforce logos (in various forms) are trademarks of salesforce.com, Inc.
+
+- OASIS, SAML and Security Assertion Markup Language are trademarks of OASIS, the open standards consortium where the <a href="http://saml.xml.org/saml-specifications" target="_blank">SAML specification</a> is owned and developed. SAML is a copyrighted © work of OASIS Open. All rights reserved.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/copyright/index.md:61]:
```suggestion
- OASIS, SAML, and Security Assertion Markup Language are trademarks of OASIS, the open standards consortium where the <a href="http://saml.xml.org/saml-specifications" target="_blank">SAML specification</a> is owned and developed. SAML is a copyrighted © work of OASIS Open. All rights reserved.
```

File: packages/@okta/vuepress-site/copyright/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+- Spotify and the Spotify logos (in several variants) are registered trademarks or trademarks of Spotify AB.
+
+- Vue.js and the Vue.js logo are trademarks or registered trademarks of Evan You
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/copyright/index.md:65]:
```suggestion
- Vue.js and the Vue.js logo are trademarks or registered trademarks of Evan You.
```

File: packages/@okta/vuepress-site/copyright/index.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+- Node.js and its wordmarks in plain text or logo form are registered trademarks or trademarks of the Node.js community.
+
+- OpenID® is a trademark (registered in numerous countries) of the OpenID Foundation.
+
+- Paypal, and the Paypal logo are registered trademarks or trademarks of Paypal, Inc.
+
+- The PHP logo is available under a Creative Commons Attribution-Share Alike 4.0 International license.
+
+- Salesforce, and the Salesforce logos (in various forms) are trademarks of salesforce.com, Inc.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/copyright/index.md:59]:
```suggestion
- Salesforce and the Salesforce logos (in various forms) are trademarks of salesforce.com, Inc.
```

**Review by susanharper-okta — APPROVED** [review]
couple edits - approving to not hold you up.

File: packages/@okta/vuepress-site/copyright/index.md (hunk 1 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/copyright/index.md:25]: Fine with me. Changed

File: packages/@okta/vuepress-site/copyright/index.md (hunk 2 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/copyright/index.md:39]: fixed

File: packages/@okta/vuepress-site/copyright/index.md (hunk 3 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/copyright/index.md:55]: fixed

File: packages/@okta/vuepress-site/copyright/index.md (hunk 4 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/copyright/index.md:61]: fixed

File: packages/@okta/vuepress-site/copyright/index.md (hunk 5 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/copyright/index.md:65]: fixed

File: packages/@okta/vuepress-site/copyright/index.md (hunk 6 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/copyright/index.md:59]: fixed

**Pull request merged**